### PR TITLE
Fixed import in loader.go

### PR DESCRIPTION
**Pull request by surajnarwade**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: cli/compose/loader/loader.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/Sirupsen/logrus"
```
Comment (thaJeztah) [inline cli/compose/loader/loader.go]: This doesn't look correct; the repository was renamed from `Sirupsen/logrus` to `sirupsen/logrus` (see https://github.com/docker/cli/commit/a0113c3a448cb1ed875f2116eb8436ee8bd97bfc, and https://github.com/docker/cli/pull/424)

The correct import path is now with a lowercase `s`: https://github.com/sirupsen/logrus/issues/553

**Comment by thaJeztah** [comment]
Closing per my comment above, but feel free to comment after that

**Comment by surajnarwade** [comment]
@thaJeztah , URL looks like https://github.com/Sirupsen/logrus and it's breaking the code
(S is capital)

**Comment by surajnarwade** [comment]
@thaJeztah my bad, I have not seen that issue before but somehow that is breaking my kompose code

**Comment by vdemeester** [comment]
@surajnarwade the problem is that with the rename, once a dependency has update logrus to reference it with the lowercase `s`, all dependencies should update too. This mean `kubernetes/kompose`, `docker/libcompose`, etc 👼

**Comment by surajnarwade** [comment]
@vdemeester thanks for clearing thing

**Pull request closed without merging**